rotkehlchen/tests/conftest.py

```diff
@@ -243,6 +243,31 @@ def vcr_config() -> dict[str, Any]:
     }
 
 
+def get_branch_distance(branch1: str, branch2: str) -> int:
+    """Get the distance between branch1 in branch2 in number of commit"""
+    merge_base = check_output(
+        args=f'git merge-base {branch1} {branch2}',
+        shell=True,
+    ).decode('utf-8').strip()
+    distance_to_branch1 = int(check_output(
+        args=f'git rev-list --count {merge_base}..{branch1}',
+        shell=True,
+    ).decode('utf-8').strip())
+    distance_to_branch2 = int(check_output(
+        args=f'git rev-list --count {merge_base}..{branch2}',
+        shell=True,
+    ).decode('utf-8').strip())
+    return distance_to_branch1 + distance_to_branch2
+
+
+def find_closest_branch(target_branch: str) -> str:
+    """Find the branch among develop and bugfixes that is closer to target_branch"""
+    all_branches = ('develop', 'bugfixes')
```

Comment:
shouldn't master also work here? So if it's a feature branch on master when we make a release to fix something shuldn't it be able to pick master?